gbpservice/nfp/configurator/drivers/firewall/asav/asav_fw_driver.py

```diff
@@ -287,7 +287,7 @@ def configure_interfaces(self, context, resource_data):
         """
 
         mgmt_ip = resource_data['mgmt_ip']
-
+        import time;time.sleep(15)
```

Comment:
This was added as a temp fix not to delay the build for demo. We have a retry logic currently in the code. We will check it there instead of delay.